indexers/entities/entities_tests.js

```diff
@@ -0,0 +1,10 @@
+import getBlock from './entities.js';
+import testBlock from './test_block.js';
+
+describe('entities.js indexer', () => {
+
+    it('accepts data with an empty namespace', () => {
+        getBlock(testBlock);
+        // successful if it does not throw an error
+    });
+});
```

Comment:
How to run this test locally?

Reply:
There's no harness for running it yet. I know that's odd. I included the test to move toward having tests and so that the PR makes a complete training case.

Reply:
Ok, got it!